source/features/linkify-labels-on-dashboard.tsx

```diff
@@ -9,11 +9,12 @@ import features from '.';
 
 function init(): void {
 	const labelClass = [
-		'.js-recent-activity-container :not(a) > .IssueLabel', // Recent activity
-		'.js-all-activity-header + div :not(a) > .IssueLabel' // Newsfeed
+		'.js-recent-activity-container :not(a) > .IssueLabel:not(.rgh-linkfied-label)', // Recent activity
+		'.js-all-activity-header + div :not(a) > .IssueLabel:not(.rgh-linkfied-label)' // Newsfeed
 	].join();
 	observe(labelClass, {
 		add(label) {
+			label.classList.add('rgh-linkfied-label');
```

Comment:
is this needed since we do :not(a)

Reply:
That's right. You can drop that and leave a comment so we avoid having this conversation again.